src/electionguard/key_ceremony_mediator.py

```diff
@@ -225,15 +195,6 @@ def _receive_election_public_key(self, public_key: ElectionPublicKey) -> None:
         """
         self._election_public_keys[public_key.owner_id] = public_key
 
-    def _all_election_public_keys_available(self) -> bool:
```

Comment:
Is the removal of this function needed since it is not using aux keys.  Just not sure if there are needs for this.

Reply:
Originally there was
`all_public_keys_received` which relied on `_all_election_public_keys_available` and `_all_auxiliary_public_keys_available`

Now it's just `all_guardian_keys_received` since we only need one keyset.